metricbeat/module/jolokia/jmx/jmx.go

```diff
@@ -69,16 +62,19 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	http.SetMethod("POST")
 	http.SetBody(body)
 
+	log := logp.NewLogger(metricsetName)
+
 	if logp.IsDebug(metricsetName) {
-		debugf("The body for POST requests to jolokia host %v is: %v",
-			base.HostData().Host, string(body))
+		log.Debugw("Jolokia request body",
```

Comment:
@andrewkroh Is this the expected use of the new logging (first time user ;-) ).

Reply:
You can omit `host` if you add it to the logger's context.

Reply:
done